common/system.h

```diff
@@ -1206,6 +1206,16 @@ class OSystem : Common::NonCopyable {
 	 */
 	virtual float getHiDPIScreenFactor() const { return 1.0f; }
 
+	/**
+	 * Return the values for the safe area insets.
+	 * Returns the values needed for the  padding to make sure any graphical element
+	 * does not interefere with any system UI elements such as the notch or home indicator
+	 * on mobile devices.
+	 *
+	 * @return The values for the top inset.
+	 */
+	virtual void getSafeAreaInsets(int &l, int &r, int &t, int &b) const { l = 0; r = 0; t = 0; b = 0; }
```

Comment:
One thing could be to return a Common::Rect here instead, but that leads to inclusion of that header file. Do we want that?

Reply:
An inset is not a Rect but more or less the opposite.
That would lead to the creation of invalid rectangles.